lib/Parse/Lexer.cpp

```diff
@@ -1688,15 +1724,21 @@ void Lexer::lexImpl() {
   case '&': case '|': case '^': case '~': case '.':
     return lexOperatorIdentifier();
 
+  case '_': case 'e':
```

Comment:
Special-casing `e` and `_` is adequate to test the feel of these features, but it doesn't tell us much about how feasible parsing the full feature—which would permit an arbitrary-length series of identifier characters in front of a string literal—would be. How I envision this actually working is that:

* The lexer scans an identifier, then (perhaps around [here](https://github.com/johnno1962d/swift/blob/master/lib/Parse/Lexer.cpp#L574)) checks if the next character is a quote mark. If so:
  * It parses the just-scanned "identifier" as a set of modifiers. (For a first cut, this "parsing" might just take the first or last character of the identifier as a `char`, but eventually it would probably scan through the identifier, setting/clearing/incrementing flags in a `StringLiteralModifier` struct/class.)
  * It passes the parsed modifiers to `lexStringLiteral()` and returns, bypassing the rest of the identifier lexing.

Does that seem like a feasible approach to you? Is it something you could incorporate into the prototype?